src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.csproj

```diff
@@ -19,6 +20,7 @@
     <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsUnix)' == 'true'">true</GeneratePlatformNotSupportedAssembly>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetGroup)' == 'netcore50'">
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <PackageTargetFramework>netstandard1.4</PackageTargetFramework>
```

Comment:
I started converting this to "netstandard1.4" as the TargetGroup, and the PInvokeChecker got angry, looks like it has a dedicated list of supported values: https://github.com/dotnet/buildtools/blob/5c97056108577de68c70d8ba436b111742e7ed0e/src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.targets#L20

All of the currently used TargetGroup values:
```
jbarton@jsb-ubuntu2:~/git/bartonjs/corefx/src$ find . -name \*.builds -exec grep TargetGroup {} \; | sort -u
      <TargetGroup>net451</TargetGroup>
      <TargetGroup>net45</TargetGroup>
      <TargetGroup>net461</TargetGroup>
      <TargetGroup>net462</TargetGroup>
      <TargetGroup>net463</TargetGroup>
      <TargetGroup>net46</TargetGroup>
      <TargetGroup>netcore50</TargetGroup>
      <TargetGroup>netcore50aot</TargetGroup>
      <TargetGroup>netstandard1.0</TargetGroup>
      <TargetGroup>netstandard1.1</TargetGroup>
      <TargetGroup>netstandard13aot</TargetGroup>
      <TargetGroup>netstandard15aot</TargetGroup>
```

So, while I agree in principle, it seems like a bit of a bigger change to add support for that; and that today might not be the best of days to do it.